### PR TITLE
Bug 1713084 - Glean Plugin: Skip buildinfo generation for libraries

### PR DESCRIPTION
**Pull request by badboy**
Requires https://github.com/mozilla/glean_parser/pull/341

---

PR against `new-event-extras`, this will automatically rebase when PR ~~#1567~~ #1603 lands.

### CODE REVIEW

File: gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy [hunk 1]
```diff
@@ -76,7 +76,7 @@ if found_version != expected_version:
             'pip',
             'install',
             '--upgrade',
-            'git+ssh://git@github.com/mozilla/glean_parser@missing-event-imports#egg=glean_parser'
+            'git+ssh://git@github.com/badboy/glean_parser@skip-buildinfo-generation#egg=glean_parser'
```
Comment (Dexterp37) [inline gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy:79]: Reminder: restore this before merging

File: gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy [hunk 2]
```diff
@@ -416,7 +424,7 @@ except:
 
                 // Setup a miniconda environment. conda is used because it works
                 // non-interactively on Windows, unlike the standard Python installers
-                conda "Miniconda3", "Miniconda3-${MINICONDA_VERSION}", "64", ["git+ssh://git@github.com/mozilla/glean_parser@missing-event-imports#egg=glean_parser"]
+                conda "Miniconda3", "Miniconda3-${MINICONDA_VERSION}", "64", ["git+ssh://git@github.com/badboy/glean_parser@skip-buildinfo-generation#egg=glean_parser"]
```
Comment (Dexterp37) [inline gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy:427]: Reminder: restore this before merging

**Review by Dexterp37 — APPROVED** [review]

**Comment by Dexterp37** [comment]
> PR against `new-event-extras`, this will automatically rebase when PR #1567 lands.

That PR was closed: should we rebase against `main`?

**Comment by badboy** [comment]
> > PR against `new-event-extras`, this will automatically rebase when PR #1567 lands.
> 
> That PR was closed: should we rebase against `main`?

Meant https://github.com/mozilla/glean/pull/1603 of course, thanks GitHub autocompletion...

**Review by mdboom — APPROVED** [review]

**Pull request closed without merging**